cms/static/js/collections/asset.js

```diff
@@ -37,8 +37,7 @@ define([
             return PagingCollection.prototype.parse.call(this, response, options);
         },
 
-        /* jshint unused:false */
-        parseState: function (response, queryParams, state, options) {
+        parseState: function(response, queryParams, state, options) {  // eslint-disable-line no-unused-vars
```

Comment:
I originally wanted to make this PR only changing jshint directives to eslint ones, but touching some of these lines made the diff-quality checker fail my build, so I had no choice but to fix them :/ There's probably several thousand lint issues in these files I haven't fixed.

But yes - to your question - the autofixer will fix space-before-function-paren.